docs/csharp/roslyn-sdk/syntax-visualizer.md

```diff
@@ -122,7 +122,7 @@ Inspect the symbol corresponding to any declared type, method, property. Select
 
 The above VB examples can be easily replicated in C#. Type `using C = System.Console;` in place of `Imports C = System.Console` for the alias. The preceding steps in C# yield identical results in the visualizer window.
 
-Semantic inspection operations are only available on nodes. They are not available on tokens or trivia. Not all nodes have interesting semantic information to inspect. When a node doesn't have interesting semantic information, clicking on **View * Symbol (if any)** shows a blank property grid.
+Semantic inspection operations are only available on nodes. They are not available on tokens or trivia. Not all nodes have interesting semantic information to inspect. When a node doesn't have interesting semantic information, clicking on **View \* Symbol (if any)** shows a blank property grid.
```

Comment:
> When a node doesn't have interesting semantic information, clicking on **View \* Symbol (if any)** shows a blank property grid

Independent of the formatting change, I was not able to connect this text with the image. Can someone help me with that?

Reply:
The previous paragraphs describe that these are on the context menu for any syntax item in the visualizer. I think it's fine as is.